src/components/shared/img-with-placeholder/index.js

```diff
@@ -32,6 +42,47 @@ const Placeholder = styled.div`
   }
 `
 
+const StyledImg = styled.img`
+  object-fit: ${props => props.objectFit || 'none'};
+  ${props => ({
+    height: props.height,
+    width: props.width,
+  })};
+`
+
+const FallbackObjectFitImg = styled.div`
+  background-size: ${props => props.objectFit};
+  background-repeat: no-repeat;
+  background-position: center center;
+  ${props => ({
+    height: props.height || '100%',
+    width: props.width || '100%',
+  })};
+  ${props => {
+    const sortedImageSet = _.sortBy(props.imageSet, ['width']) // ascending
+    let bgImageCssString = ''
+    const imagesCount = sortedImageSet.length
+    for (let i = 0; i < imagesCount; i += 1) {
+      const currentImage = sortedImageSet[i]
+      if (i === 0) {
+        const nextImage = sortedImageSet[i + 1]
+        bgImageCssString += `
+          @media (max-width: ${nextImage.width - 1}px) {
```

Comment:
I made a mistake. Select which image should according to the width of image layout width rather than screen width. I'll fix this on Monday.